src/openrct2/game.c

```diff
@@ -1083,10 +1083,12 @@ bool game_load_save(const utf8 *path)
     bool result = false;
     if (extension_type == FILE_EXTENSION_SV6) {
         result = game_load_sv6_path(path);
+        if (result)
```

Comment:
Yeah, this also fixes that. the gFirstTimeSaving being true for sv4s will force the first save to show a "save as" dialog, while sv6 would mark it false saying it's okay to overwrite the loaded file